src/nodedef.cpp

```diff
@@ -406,6 +406,7 @@ void ContentFeatures::reset()
 	move_resistance = 0;
 	liquid_move_physics = false;
 	post_effect_color_shaded = false;
+	climb_factor = 1.0f;
```

Comment:
Please move this below `climbable` or move the initialization to `nodedef.h` (L391).